fili-core/src/main/java/com/yahoo/bard/webservice/data/metric/protocol/ProtocolSupport.java

```diff
@@ -52,9 +59,26 @@ public ProtocolSupport(
     public ProtocolSupport(
             Collection<Protocol> protocols,
             Set<String> blacklist
+    ) {
```

Comment:
in the javadoc for this method add:

Name is defaulted to empty string, indicating that this protocol support will not publish metadata for the base metric.